package-lock.json

```diff
@@ -32400,62 +32160,6 @@
       "dev": true,
       "optional": true
     },
-    "@nrwl/nx-darwin-x64": {
-      "version": "15.8.6",
-      "resolved": "https://registry.npmjs.org/@nrwl/nx-darwin-x64/-/nx-darwin-x64-15.8.6.tgz",
-      "integrity": "sha512-h9/JULzPZTpt6oNpKMZLc1NGDu+CLyx91c8DJUh/hH0Zh/7dS9LFxe9jWeFIdh18iAu7ZAoktK2KJ5YhOrUYhA==",
-      "dev": true,
-      "optional": true
-    },
-    "@nrwl/nx-linux-arm-gnueabihf": {
-      "version": "15.8.6",
-      "resolved": "https://registry.npmjs.org/@nrwl/nx-linux-arm-gnueabihf/-/nx-linux-arm-gnueabihf-15.8.6.tgz",
-      "integrity": "sha512-Yp/YjzcIHW+OW4revPRZIt0Px9cKRsOL69FPLlYSxWuR/PD9SPeXWcbo3pKkjnIWIjOL2YT8z5cHiQ3bV1NVfw==",
-      "dev": true,
-      "optional": true
-    },
-    "@nrwl/nx-linux-arm64-gnu": {
-      "version": "15.8.6",
-      "resolved": "https://registry.npmjs.org/@nrwl/nx-linux-arm64-gnu/-/nx-linux-arm64-gnu-15.8.6.tgz",
-      "integrity": "sha512-b+OenpPhhxqgaG6EFHRLfVGtAU4+UbKqOhv7DLLh5P7tX3RAQAtyrT6tVkfDRFYl6kgEme/I5ZrevcbaGyDO+w==",
-      "dev": true,
-      "optional": true
-    },
-    "@nrwl/nx-linux-arm64-musl": {
-      "version": "15.8.6",
-      "resolved": "https://registry.npmjs.org/@nrwl/nx-linux-arm64-musl/-/nx-linux-arm64-musl-15.8.6.tgz",
-      "integrity": "sha512-F9D8moy+lfJQhVrZoY54vDwpigBgxQy4HB9PRmc6Ln9mIk3ouOvKNC99OjUYEO+ensHr9eMpsbghsRCjod//uw==",
-      "dev": true,
-      "optional": true
-    },
-    "@nrwl/nx-linux-x64-gnu": {
-      "version": "15.8.6",
-      "resolved": "https://registry.npmjs.org/@nrwl/nx-linux-x64-gnu/-/nx-linux-x64-gnu-15.8.6.tgz",
-      "integrity": "sha512-an0zD6lKpblexazKssFvcfOx7BuGutwlrzmwScxISPXj5+ly99u+sYclDg2P56YRHYXIuYGBK0c0VWaJ91QIcw==",
-      "dev": true,
-      "optional": true
-    },
-    "@nrwl/nx-linux-x64-musl": {
-      "version": "15.8.6",
-      "resolved": "https://registry.npmjs.org/@nrwl/nx-linux-x64-musl/-/nx-linux-x64-musl-15.8.6.tgz",
-      "integrity": "sha512-LOurlSuLf9LWdjvpHHIsHC0auxgMVrkeOFFCUJ3oVv/dN4uZ0vuNG98XM2E7fPDXDacBZIyKdx34KQlmFfBHsA==",
-      "dev": true,
-      "optional": true
-    },
-    "@nrwl/nx-win32-arm64-msvc": {
-      "version": "15.8.6",
-      "resolved": "https://registry.npmjs.org/@nrwl/nx-win32-arm64-msvc/-/nx-win32-arm64-msvc-15.8.6.tgz",
-      "integrity": "sha512-MvH84nLv1tdM96z92abeQd+tguY/zKC22AFHek9PSR1StUQzwwPu6rR7XDn3mggwnkLm11jTUXlk7wdbE5sldQ==",
-      "dev": true,
-      "optional": true
-    },
-    "@nrwl/nx-win32-x64-msvc": {
-      "version": "15.8.6",
-      "resolved": "https://registry.npmjs.org/@nrwl/nx-win32-x64-msvc/-/nx-win32-x64-msvc-15.8.6.tgz",
-      "integrity": "sha512-P0Sb4HJCeoeTvPFdUMKljRUIjzso0go36cn1Zpl+Z5CG/nbOvLlbnzh6rg15SRNu9OLWTHNPtyQIvKxjqEDoxg==",
-      "dev": true,
-      "optional": true
-    },
```

Comment:
This is interesting. These shouldn't have been removed. Thats why they don't get installed on the CI

Reply:
let me try to remove the lockfile and node_modules and regenerate it, maybe something went wrong on my end

Reply:
https://github.com/kordis-leitstelle/kordis/actions/runs/4439289647/jobs/7791543255 well... idk why this is so flocky, but it seems something went wrong when we migrated or something. Idk if that might be related to the original issue. Hope the insights you got here helped! @Cammisuli